tests/integration/test_wazuh_db/test_wazuh_db.py

```diff
@@ -77,3 +116,23 @@ def test_wazuh_db_create_agent(configure_sockets_environment, connect_to_sockets
                            "stage": "Syscheck - Agent does not exits yet"}]}
     test_wazuh_db_messages(configure_sockets_environment, connect_to_sockets_module, test['test_case'])
     assert os.path.exists(os.path.join(WAZUH_PATH, 'queue', 'db', "999.db"))
+
+
+def test_wazuh_db_chunks(configure_sockets_environment, connect_to_sockets_module, pre_insert_agents):
+    """Check that commands by chunks work properly when agents amount exceed the response maximum size""" 
+
+    def test_chunk_command(command):
+        receiver_sockets[0].send(command, size=True)
+        response = receiver_sockets[0].receive(size=True).decode()
+    
+        status = response.split(" ",1)[0]
+        assert status == 'due', 'Failed chunks check on < {} >. Expected: {}. Response: {}'\
+               .format(command, 'due', status)
+   
+    #Check get-all-agents chunk limit
+    test_chunk_command(f'global get-all-agents last_id 0')
+    #Check get-agents-by-keepalive chunk limit
+    test_chunk_command(f'global get-agents-by-keepalive condition > -1 last_id 0')
+    #Check sync-agent-info-get chunk limit
+    test_chunk_command(f'global sync-agent-info-get last_id 0')
+    
```

Comment:
```suggestion
def test_wazuh_db_chunks(configure_sockets_environment, connect_to_sockets_module, pre_insert_agents):
    """Check that commands by chunks work properly when agents amount exceed the response maximum size""" 

    def send_chunk_command(command):
        receiver_sockets[0].send(command, size=True)
        response = receiver_sockets[0].receive(size=True).decode()
    
        status = response.split(" ", 1)[0]
        assert status == 'due', 'Failed chunks check on < {} >. Expected: {}. Response: {}'\
               .format(command, 'due', status)
   
    # Check get-all-agents chunk limit
    send_chunk_command(f'global get-all-agents last_id 0')
    # Check get-agents-by-keepalive chunk limit
    send_chunk_command(f'global get-agents-by-keepalive condition > -1 last_id 0')
    # Check sync-agent-info-get chunk limit
    send_chunk_command(f'global sync-agent-info-get last_id 0')
```

Only test functions can begin with `test_` as this is what **pytest** will parse from the file. I suggest renaming the inner function.